### DISCUSSION TITLE
Importing directly from module entry-points is not allowed & making @radix-vue/nuxt package

### DISCUSSION THREAD

**Discussion by alexisdechiara** [unovue/reka-ui#1128] [General]
Hi, for a few days now I've been getting the error `Importing directly from module entry-points is not allowed` when launching my project under Nuxt and I didn't see any fix on the web until I went on Discord where someone named Zusor solved the same problem I was having.

If you install the package with the Nuxt installer `nuxt module add radix-vue` then the whole package will be added to Nuxt.config, but in the documentation you have to put `radix-vue/nuxt` in the modules and not `radix-vue`. So you'll understand that to solve the problem you just need to add "/nuxt" and it's fixed, but on the [Nuxt module page of radix-vue](https://nuxt.com/modules/radix-vue) it's not specified to make this change, which can lead people like Zusor and me to misunderstand.

For this reason, it would be preferable to have a `@radix-vue/nuxt' package separately so that the error no longer occurs when installing via the Nuxt installer and other package manager. In addition, it would be interesting to modify the Nuxt module page so that it is identical to the one in the doc.

<img width="540" alt="Arc_2024-07-16_10-01-42" src="https://github.com/user-attachments/assets/dae87e15-5056-40a5-9b8c-317f11d9894f">

**Comment by zernonia** [comment]
Hey @alexisdechiara !! Thanks for bringing this up. I'll check whether Nuxt module allow custom 'module name' when installing via `nuxt module add`
